okonomiyaki/platforms/_pep425_impl.py

```diff
@@ -76,7 +76,8 @@ def get_abi_tag():
         if get_flag('WITH_PYMALLOC',
                     lambda: impl == 'cp',
                     warn=(impl == 'cp')):
-            m = 'm'
+            if sys.version_info < (3, 8):
```

Comment:
I'm guessing that the fallback value is used for finding the flag WITH_PYMALLOC for Python 3.8 which is giving the incorrect value. It seems like ideally there would be a way to find out if the flag is really being used or not.

Reply:
The get_flag function is performing the check for the use of pymalloc already bit is no `m` indication for Python 3.8 and above even if the runtime was compiled with pymalloc or not.